### ISSUE TITLE
Issue retrieving link data from a link shared to facebook

### ISSUE THREAD

**Issue by rklus16** [BranchMetrics/cordova-ionic-phonegap-branch-deep-linking#556]
Not reporting this as a bug but just looking to see if there's something I'm missing.

I'm sharing a branch link via the sharesheet to facebook.  On ios, clicking the post in facebook it redirects to the app but without any data:
```{"+clicked_branch_link":false,"+is_first_session":false}```

from an android device, the same facebook post loads the app and displays the data correctly. 

I can take that same link and send it over iMessage and works perfect in ios?

Any ideas on what's going on here? thanks.

**Comment by DaniShalash** [comment]
Same issue here, I've been struggling for days now and Branch doesn't seem to work on iOS.

**Comment by csalmi-branch** [comment]
@rklus16 and @DaniShalash are you still experiencing this issue? If so, could you provide a bit more information about your configuration? What version of the SDK are you using, etc. 

Could you also confirm that you have the same keys (either test or live) in both the link creation and in the SDK?

Our engineering team will be taking a look into this issue.

**Comment by rklus16** [comment]
@csalmi-branch Yes, I'm still getting the same results. Using the sharesheet on iOS, I can copy the link into notes and works perfectly into the app. If I share that same link in FB then click the link, it opens the app but without any data.

Ionic 4.12.0
branch-cordova-sdk 3.1.6

**Comment by DaniShalash** [comment]
@csalmi-branch @rklus16 
I got it working now.
The strange thing is the only thing I did was putting my app URI scheme before Facebook SDK's generated one in URL Types in info.plist
Facebook SDK inserts a scheme like this "fb<fb_app_id>" e.g. "fb423904823023"

ionic 3.9.2
branch-cordova-sdk 3.1.6

**Comment by rklus16** [comment]
@DaniShalash are you referring to this? This is the order mine are in.
<img width="880" alt="Screen Shot 2019-04-03 at 9 10 49 AM" src="https://user-images.githubusercontent.com/3171887/55486202-89ddbb00-55f1-11e9-9c5f-7e8ea01c402e.png">
